vehicle/saic/api.go

```diff
@@ -50,17 +55,7 @@ func NewAPI(log *util.Logger, identity *Identity) *API {
 	return v
 }
 
-/*
-func (v *API) printAnswer() {
-	v.Logger.DEBUG.Printf("SOC:%d ", v.request.Result.ChrgMgmtData.BmsPackSOCDsp)
-	v.Logger.DEBUG.Printf("GUN:%d ", v.request.Result.RvsChargeStatus.ChargingGunState)
-	v.Logger.DEBUG.Printf("Chrg State:%d ", v.request.Result.ChrgMgmtData.BmsChrgSts)
-	v.Logger.DEBUG.Printf("Mileage:%d ", v.request.Result.RvsChargeStatus.Mileage)
-	v.Logger.DEBUG.Printf("Range:%d ", v.request.Result.RvsChargeStatus.FuelRangeElec)
-}
-*/
-
-func (v *API) doRepeatedRequest(url string, event_id string) error {
+func (v *API) doRepeatedRequest(baseUrl string, path string, event_id string) error {
```

Comment:
Do we really need the baseUrl as parameter if v already has access to the identity's baseurl?

Reply:
I will have a look tonight and see if I can simplify.